### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by cyberpastey**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by cyberpastey — COMMENTED** [review]
The request should be accepted. It is seamless

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```
Comment (MayorC) [inline index.html:47]:
```Get a extra table in this table
  </tr>
```

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (MayorC) [inline index.html:50]:
```create  new header here
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (cyberpastey) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again and get new rewards 🧑‍💻 🤖!</h2>
```

Comment (cyberpastey) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again and get new rewards 🧑‍💻 🤖!</h2>
```

**Pull request merged**